vars/getFunctionalTags.groovy

```diff
@@ -49,10 +49,71 @@ Map call(Map kwargs = [:]) {
         requested_tags = requested_tags.trim()
     }
 
+    // Add any Skip-list tags to each requested tag
+    /* groovylint-disable-next-line UnnecessaryGetter */
+    List skipped = getSkippedTests()
+    if (skipped) {
+        String tags = ''
+        for (group in requested_tags.split(' ')) {
+            tags += group + ',-' + skipped.join(',-') + ' '
+        }
+        requested_tags = tags.trim()
+    }
+
```

Comment:
Is this right? The idea is maybe we have a commit message like:

```
Features: foo
```
And also in that commit message (or some other mechanism) we have
```
Skip-list: test_to_skip:DAOS-1234
```

Simplified to ignore pr, daily, etc. we end up running
```
foo,-test_to_skip
```

Reply:
Yes, that's the goal.